runner.rb

```diff
@@ -7,4 +7,6 @@
 	new_game.draw
 end
 
+puts "Test for PR"
```

Comment:
Hey im not sure why this is here.  Do we need to add a puts in this file for anything other than driver code?